cassandane/Cassandane/TestUser.pm

```diff
@@ -62,6 +62,54 @@ has jmap => (
     }
 );
 
+has jmaptester => (
+    is => 'ro',
+    lazy => 1,
+    default => sub {
+        my ($self) = @_;
+
+        unless ($self->instance->{config}->get_bit('httpmodules', 'jmap')) {
+            Carp::croak("User JMAP::Tester requested, but jmap httpmodule not enabled");
+        }
+
+        my %arg = $self->_common_http_service_args->%*;
+
+        my $host = $arg{host};
+        my $port = $arg{port};
+
```

Comment:
I know you'd like to fix it all to be better, but at least for now to be compatible I think you should do this before creating a new JMAP::Tester object:
```
        # Send jmap traffic to STDERR like Mail::JMAPTalk above does
        local $ENV{JMAP_TESTER_LOGGER} = 'HTTP:-2'
            unless exists $ENV{JMAP_TESTER_LOGGER};
```

That way we get roughly the same debug output at the right point in the tests

Reply:
Done.